sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -61,21 +61,15 @@ trait TimeZoneAwareExpression extends Expression {
 @ExpressionDescription(
   usage = "_FUNC_() - Returns the current date at the start of query evaluation.",
```

Comment:
Is it good to specify the time zone in this usage text?

Reply:
Updated docs (and comments) of the expression and the current_date() function.